pipes/config.yaml

```diff
@@ -192,6 +192,27 @@ vphaser_max_bins: 10
 # allele freq > 0.005. If false, no filtering is performed.
 vcf_merge_naive_filter: false
 
+# Preprocessing method for reads for de novo assembly
+assembly_preproc_method: 'subsamp'
+
+# Method of de novo assembly
+assembly_de_novo_method: 'trinity'
+
+# Method of choosing the reference to use for scaffolding
+refchoice_for_scaffold_method: 'common'
```

Comment:
Could params be grouped hierarchically, by stage?

assembly:
    - preproc_method: 'subsamp'
    - de_novo_method: 'trinity'
scaffolding:
    - reference_selection_method: 'common'
    - scaffolding_method: 'viral-ngs-scaffolder'
finishing:
    - gapfill_method: 'gap2k'

etc